prototype_source/fx_graph_mode_ptq_static.rst

```diff
@@ -54,14 +54,9 @@ Advantages of FX Graph Mode Quantization are:
   
 We’ll start by doing the necessary imports, defining some helper functions and prepare the data.  
 These steps are identitcal to `Static Quantization with Eager Mode in PyTorch <https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html>`_.       
-  
-Download dataset: 
-  
-.. code:: 
-  
-    wget https://s3.amazonaws.com/pytorch-tutorial-assets/imagenet_1k.zip 
-  
-and unzip to `data` folder.   
+
+Get a sample imagenet dataset imagenet_1k.zip unzip to `data` folder. (The download url is removed due to legal issues. We'll write down instructions on how to get imagenet_1k.zip data later.)
```

Comment:
How about:

To avoid requiring the download of the ImageNet dataset, we provide a pointer to download a pretrained model as a starting point.

Reply:
thanks, I copied the wording from the eager mode tutorial, please let me know if works